src/style.css

```diff
@@ -2,0 +1,26 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

Comment:
Ensure that each BEM block has its own separate file as per the task requirements. This helps in maintaining modularity and reusability of components.